.github/workflows/workflows.yml

```diff
@@ -3,7 +3,7 @@ name: standard tests
 on:
   pull_request:
   push:
-    branches: [main, development]
+    branches: [main, development, tests]
```

Comment:
I guess this branch will be deleted at some point. Maybe add a test only for pull requests on development and main? Then the tests can be run locally before opening the PR after which the test is run in Github Actions.